src/main/java/org/selenium/pages/CartPage.java

```diff
@@ -0,0 +1,46 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.selenium.ojb.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class CartPage extends BasePage {
+    private By cartButton = By.xpath("//a[@title='View my shopping cart']");
+    private By waitContainer = By.xpath("//div[@id='center_column']");
+    private By cartTotal = By.xpath("//span[@id='total_price']");
+    private By rows = By.xpath("//table/tbody/tr");
+    private List<Product> cartList = new ArrayList<>();
+
+    public List<Product> getCartList() {
+        driver.findElement(cartButton).click();
+        wait.until(visibilityOfElementLocated(waitContainer));
+        for (WebElement row : driver.findElements(rows)) {
+            String attribute = row.findElement(By.xpath("./td[2]/small/a")).getText();
+            String name = row.findElement(By.xpath("./td[2]//p/a")).getText();
+            String color = attribute.substring(attribute.indexOf("Color : ") + 8, attribute.indexOf(","));
+            String size = attribute.substring(attribute.indexOf("Size : ") + 7);
+            String price = row.findElement(By.xpath("./td[6]")).getText();
+            String priseSub = price.substring(price.indexOf("$") + 1);
+            double p = Double.valueOf(priseSub);
+            cartList.add(getProduct(name, color, size, p));
+        }
+        return cartList;
+    }
+
+    private Product getProduct(String name, String color, String size, double price) {
+        Product product = new Product(name, color, size, price);
+        return product;
+    }
```

Comment:
I don't see the need of this method. It makes creation of new instance. 
The line 30 can be written in the next way `cartList.add(new Product(name, color, size, p));`